qemu/tests/qemu_guest_agent.py

```diff
@@ -2312,7 +2312,7 @@ def _guest_cmd_run(guest_cmd, cmd_args=None, env_qga=None,
     def _action_before_fsfreeze(self, *args):
         session = self._get_session(self.params, None)
         if self.params.get("os_type") == "linux":
-            session.cmd("restorecon -Rv /", timeout=180)
+            session.cmd("restorecon -Rv /", timeout=360)
```

Comment:
The original problem is only under `/mnt`, so I think here we can limit to restorecon in the `/mnt` dir to reduce time. Can you try?

Reply:
Hi @PaulYuuu 
Yes, it makes sense, thanks for the review. 
Updated.

JOB ID     : b3dc2ead62b9c9dc7675d7a82a11d59bf9f9965a
JOB LOG    : /root/avocado/job-results/job-2024-02-09T09.29-b3dc2ea/job.log
 Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.aarch64.page_64k.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_fsfreeze.arm64-pci: STARTED
 Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.aarch64.page_64k.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_fsfreeze.arm64-pci: PASS